src/molecules/formfields/RadioField/index.js

```diff
@@ -19,10 +20,13 @@ function RadioField(props) {
     className,
   ];
 
+  const isChecked = !isUndefined(input.checked) ? input.checked
```

Comment:
can you explain a bit why we need to update this logic? didn't know there was a bug here but not using the `input.checked` attribute.

Reply:
+1 I am not sure what the exact difference is here except for checking against undefined as well as checking for if the `input.value` matches the `radioValue`

Reply:
+1 confused as well. It looks like the original code never checked for if `input.value` was defined before it checked if the `input.value` matched the `radioValue`, so I could see that leading to some issues, but not sure what `input.checked` vs. `input.value` does. Couldn't we just use `input.checked` and `!input.checked` in the `labelClasses` for if they are selected or not?

Reply:
As discussed, because of how `react-final-form` components communicates with `athenaeum`, sometimes the `checked` attribute is not available. We will keep this as backwards compatible as possible but push towards using the `checked` value if possible.